report/generator/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
these new files are temporary, either removed when report-generator goes ESM or when it's moved to `shared/`